builtins/pcre/CMakeLists.txt

```diff
@@ -13,13 +13,19 @@ foreach(var PCRE_FOUND PCRE_VERSION PCRE_INCLUDE_DIR PCRE_PCRE_LIBRARY PCRE_LIBR
 endforeach()
 
 if(WIN32)
-  if(winrtdebug)
-    set(PCRE_POSTFIX $<$<CONFIG:Debug>:d>)
-    set(pcre_config_kind "Debug")
+  if(CMAKE_GENERATOR MATCHES Ninja)
+    if (CMAKE_BUILD_TYPE MATCHES Debug)
+      set(PCRE_POSTFIX d)
+    endif()
   else()
-    set(pcre_config_kind "Release")
+    if(winrtdebug)
+      set(PCRE_POSTFIX $<$<CONFIG:Debug>:d>)
+      set(pcre_config_kind "Debug")
+    else()
+      set(pcre_config_kind "Release")
+    endif()
+    set(pcre_config "--config")
```

Comment:
Wait, is this not in the `else()` of the `if(WIN32)` branch? Why is `winrtdebug` relevant then? I thought `if(WIN32)` is *true* for *any* ROOT build on Windows

Reply:
The code is correct, but I'll make it more clear